runtime/compiler/env/J9PersistentInfo.cpp

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+// #if defined(J9VM_OPT_JITSERVER)
+#include "control/CompilationThread.hpp"
+#include "runtime/JITClientSession.hpp"
+// #endif
+#include "env/PersistentInfo.hpp"
```

Comment:
Shouldn't this be `J9PersistentInfo.hpp` ?

Reply:
Well `PersistentInfo.hpp` includes `J9PersistentInfo.hpp` so that should be fine.
In all other instances of this include non-J9 version is used so I used it here as well.
Changing it to J9 doesn't seem to have any effect.